src/io.c

```diff
@@ -27,7 +27,10 @@ static avifResult avifIOMemoryReaderRead(struct avifIO * io, uint32_t readFlags,
 {
     // printf("avifIOMemoryReaderRead offset %" PRIu64 " size %zu\n", offset, size);
 
-    (void)readFlags;
+    if (readFlags != 0) {
```

Comment:
Joe: An alternative is to remove the readFlags parameter from avifIOReadFunc. Would you prefer that?